paper-ieee-trans.tex

```diff
@@ -312,23 +314,23 @@ \subsection{EGI--Integrated Sustainable Pan--European Infrastructure}
 
 The {\sl EGI--InSPIRE} (Integrated Sustainable Pan--European Infrastructure for
 Researchers in Europe, May 2010 -- May 2014) project \cite{cordis:egi-inspire}
-arose as the continuation of the {\sl EGEE--III} project, with the mission of establishing and
+was the continuation of the {\sl EGEE--III} project, with the objective of establishing and
 maintaining a sustainable European Grid Infrastructure, composed by a federation of National 
 Grid Initiatives and interoperable with other Grids worldwide.
-To accomplish this goal the project foreseed the 
-maintenance and development of operational tools, like the Operations Portal \cite{egi-ops}, 
-the {\sl EGI} Helpdesk \cite{ggus} or the Grid configuration database (GOCDB) \cite{gocdb}, and the 
-management of the software provisioning process \cite{mario}, to validate and distribute the software that conformed
-the production infrastructure. As a result of this process, a production--ready 
-Grid computing middleware distribution named {\sl UMD} (Unified Middleware Distribution) was
-maintained, comprised by roughly 250 software components supplied by diverse technology providers such 
+This goal would be accomplished through the
+development and maintenance of operational tools, such as the Operations Portal \cite{egi-ops}, 
+the {\sl EGI} Helpdesk \cite{ggus}, or the Grid configuration database (GOCDB) \cite{gocdb}, and the 
+management of the software provisioning process \cite{mario}, to validate and distribute the software to
+the production infrastructure. As a result, this process led to a production--ready 
+Grid computing middleware distribution named {\sl UMD} (Unified Middleware Distribution). The {\sl UMD}
+was a stack of about 250 software components from several technology providers such 
```

Comment:
Probably open with "At that time the UMD was a stack of about.." since, as it is already said below UMD is still in use

Reply:
ok